app/views/posts/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1 name='title'>Here is a list of posts for User Id: <%= @user_id%></h1>
\ No newline at end of file
+<head>
+</head>
+<body>
+  <div>
+    <%= render partial: 'users/user', locals: { user: @user } %>
+  </div>
+  <div>
+    <% @posts.each_with_index do |post, index| %>
```

Comment:
- [ ] Please use the `most_recent_comment` method you created in your Post model to render the 5 most recent comments as mentioned in the project requirements.